preprocessing/libs/vis2.py

```diff
@@ -1,5 +1,5 @@
 """Visualization utilities imported from kzpy3."""
-from utils2 import *
+from .utils2 import *
```

Comment:
Are these .imports compatible with python2 as well? It would be good if our code was back compatible.